src/logistration/RegistrationPage.jsx

```diff
@@ -192,24 +215,44 @@ class RegistrationPage extends React.Component {
 
     switch (inputName) {
       case 'email':
-        emailValid = value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i);
-        errors.email = emailValid ? '' : this.intl.formatMessage(messages['logistration.email.validation.message']);
+        emailValid = value.length >= 1;
+        if (!emailValid) {
+          emptyFields.email = [{ user_message: this.intl.formatMessage(messages['logistration.email.validation.message']) }];
+        } else {
+          emptyFields.email = [{ user_message: '' }];
```

Comment:
Can we handle all these errors using a utility function like the following?
```
  generateUserMessage = (isFieldValid, messageID) => {
    return [{ user_message: isFieldValid ? '' : this.intl.formatMessage(messages[messageID] }];
  }
```
```suggestion
        emptyFields.email = generateUserMessage(emailValid, 'logistration.email.validation.message']);
```

Reply:
Done